TestVectors/eddsa-2022/canonDocDataInt.txt

```diff
@@ -0,0 +1,8 @@
+<did:example:abcdefgh> <https://schema.org/alumniOf> "The School of Examples" .
+<urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://www.w3.org/2018/credentials#VerifiableCredential> .
+<urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://www.w3.org/ns/credentials/examples#AlumniCredential> .
+<urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33> <https://schema.org/description> "An minimum viable example of an Alumni Credential." .
```

Comment:
```suggestion
<urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33> <https://schema.org/description> "A minimum viable example of an Alumni Credential." .
```

Reply:
Another artifact?

Reply:
Artifact? Quite a while ago it was requested that all test vector be handled via ReSpec's `data-include` mechanism rather than embedded/escaping lots of JSON and other information directly into the `index.html` file. These are kept in a separate `TestVectors` directory and included via ReSpec mechanisms.

Reply:
Either I'm commenting on artifacts — indirectly managed files produced from directly managed files by CI or similar processes — in which case these indirectly managed files should be added to `.gitignore` or similar so they don't show up in the `/files` view of a given PR, *OR* I'm commenting on those directly managed files, in which case I request that my suggestions be addressed with somewhat more speed.

I believe my suggestions should generally be accepted/merged into the PR fork, but you or others may disagree with my suggestions, in which case comments in reply to my suggestions should reflect such disagreement, so that we may work to resolve misunderstanding or other basis for the disagreement, and come to agreement on appropriate change (which may include retraction of my suggestion).